Character.cs

```
@@ -0,0 +1,466 @@
+                List<string> checkWeapon = new List<string>();
+                foreach (Item weapon in Items) { checkWeapon.Add(weapon.Name.Trim().ToLower()); }
+                for (i = weapon1.Count - 1; i >= 0; i--)
+                {
+                    Weapon x = weapon1[i]; // change monster class to having two lists, or start with reviewing all weapons throughout and construct new item from weapon details
+                    if (alreadyStashed)
+                    {
+                        Console.WriteLine("You've already taken that item!");
+                        break;
+                    }
+                    else if (x.Name.Trim().ToLower() == answer)
+                    {
+
+                        x.pickUpItem(inventory, weaponInventory, 3, 0, null, x);
+                        answer = "";
+                        skip = true;
+
+                        try
+                        {
+                            if (weaponInventory.Contains(x))
+                            {
+                                Items.Remove(x);
+                            }
+
+
+                            else if (inventory.Contains(x))
+                            {
+                                Items.Remove(x);
+                            }
+                        }
+                        catch { }
+
+                        break;
+                    }
+
+
+
+                }
+                for (i = Items.Count - 1; i >= 0; i--)
+                {
+                    if (skip) { break; }
+                    Item x = Items[i];
+                    if (alreadyStashed)
+                    {
+                        Console.WriteLine("You've already taken that item!");
+                        break;
+                    }
+
+                    else if (x.Name.Trim().ToLower() == answer)
+                    {
+                        x.pickUpItem(inventory, weaponInventory, 3, 0, x);
+                        try
```

Is there any reason to having a try block with an empty catch block?

---

yes